src/app/shared/components/phone-form-control/phone-form-control.component.ts

```diff
@@ -46,4 +47,12 @@ export class PhoneFormControlComponent implements AfterViewInit, OnDestroy {
     this.destroy$.next(true);
     this.destroy$.unsubscribe();
   }
+
+  @HostListener('document:keypress')
+  public handleKeyboardEvent() {
+    const countrySearchElement = document.querySelector('.ngx-mat-tel-input-mat-menu-panel .country-search') as HTMLElement;
+    if (countrySearchElement) {
+      setTimeout(() => countrySearchElement.focus(), 200);
+    }
+  }
```

Comment:
These lines fix the issue of country search becoming unfocusable on slow typing.
Like there:
![1](https://github.com/ita-social-projects/OoS-Frontend/assets/68949838/de3d5890-ef3f-462e-8fca-56b0e655c8aa)
The approach described [here](https://github.com/tanansatpal/ngx-mat-intl-tel-input/issues/157) is used to handle every `document:keypress` event (it is used whole document event because it doesn't emit in country search but somewhere in menu container). However, you can see a little bit blink of focus events.
There how it looks now:
![2](https://github.com/ita-social-projects/OoS-Frontend/assets/68949838/79fe53fb-184c-4cfc-88b8-2b2041259023)